codegen/src/main/java/software/amazon/awssdk/codegen/naming/DefaultNamingStrategy.java

```diff
@@ -125,21 +129,42 @@ public String getVariableName(String name) {
 
     @Override
     public String getEnumValueName(String enumValue) {
-        StringBuilder builder = new StringBuilder();
+        String result = enumValue;
 
-        String sanitizedEnumValue = enumValue.replace("::", ":").replace("/", "").replace("(", "")
-                                             .replace(")", "");
+        // Special cases
+        result = result.replaceAll("IoT", "IOT")
```

Comment:
Should we treat these as customizations rather then put them in Core?

Reply:
Do we have the feature in place to customize enum names? I didn't see anything like that.

Reply:
I'm not sure, but we can add one. In general we try not to put service specific customizations in the code gen.

Reply:
That's fair - it's not good to have service-specific stuff in the generator. 

IoT in this case isn't service-specific. I'm not aware of any service using it in their enums, but they may in the future and we'd probably not notice it (generating IO_T instead).

It's just textORcsv that is service-specific in this PR. Because it's just this one example, I'd vote we follow the rule-of-three and add a customization when we have more cases in which we would use it. 

I don't think it is likely that more cases like this are going to happen. AWS service owners are warned now if their enums don't follow SCREAMING_CASE conventions. If something does happen to get through review: (a) we'll catch it before it's released and have the service owner fix their model, or (b) we won't catch it before it's released and at that point we can't change it anyway.

Reply:
Should the IoT case just be baked into the logic then. I.E. is there any value where we'd want to separate a single capital letter via underscore? If not can we just treat that as part of the previous word boundary?

Reply:
Sounds good to me.